packages/compass-telemetry/src/generic-track.ts

```diff
@@ -1,52 +1,53 @@
-import { createLogger, mongoLogId } from '@mongodb-js/compass-logging';
+import {
+  type Logger,
+  mongoLogId,
+  createLogger,
+} from '@mongodb-js/compass-logging';
```

Comment:
"Generic" track can't have non-type dependencies on platform-specific code. It's not really something we can easily catch with automation, so we should just stay alert when adding services